testing/simapp/app.go

```diff
@@ -491,9 +501,11 @@ func NewSimApp(
 	ibcRouter := porttypes.NewRouter()
 
 	// Middleware Stacks
+	maxCallbackGas := uint64(1_000_000)
```

Comment:
I wonder should we include some reasonable default value in the callbacks types pkg for this value - then that could be used here. Just thought this var seemed a little randomly placed as its not used until down below where we wrap the transfer stack, and this came to mind. We can decide on it later tho!

Reply:
Let's decide this later since we don't wanna import more things from callbacks.

Reply:
💯